assignments/callbacks.js

```diff
@@ -24,25 +24,46 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+ 
+getLength(items, arrLength => console.log(arrLength));
+
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length - 1])
 }
 
+last(items, lastItem => console.log(lastItem));
+
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  let sumOfNums = x + y;
+  return cb(sumOfNums);
 }
 
+sumNums(1, 9, numSum => console.log(numSum));
+
```

Comment:
when you're using the exact same callback for every function, it helps to define it in the file above and just pass it into the invocation instead